plugins/lighthouse-backend/README.md

```diff
@@ -80,5 +86,8 @@ You can define how often and when the scheduler should run the audits:
 ```yaml
 lighthouse:
   schedule:
-    days: 1
+    frequency:
+      hours: 12 # Default: days 1
+    timeout:
+      minutes: 30 # Default: null
```

Comment:
whenever you change the config format, you should update the according `config.d.ts` file. But to my surprise, I see that this plugin didn't even have one yet. That's actually not great at all.

Could you add one in the backend plugin folder modelled [after the catalog one](https://github.com/backstage/backstage/blob/5114713ee02dad71c22fa480fb99125a510f6e0b/plugins/catalog-backend/config.d.ts#L19)?

It can use the HumanDuration import just like that file.

Note that then the package.json file also needs to have config.d.ts mentioned in `files`, [like this](https://github.com/backstage/backstage/blob/5114713ee02dad71c22fa480fb99125a510f6e0b/plugins/catalog-backend/package.json#L105), and also have the `configSchema` field, [like this](https://github.com/backstage/backstage/blob/5114713ee02dad71c22fa480fb99125a510f6e0b/plugins/catalog-backend/package.json#L107).

It should probably be something like

```ts
import { TaskScheduleDefinitionConfig } from '@backstage/backend-tasks';
import { HumanDuration } from '@backstage/types';

export interface Config {
  lighthouse?: { 
    schedule?: HumanDuration | TaskScheduleDefinitionConfig;
  }
}
```